Userland/Utilities/shot.cpp

```diff
@@ -152,6 +154,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         warnln("Failed to encode PNG");
         return 1;
     }
+    if (edit_image) {
+        output_path = Core::DateTime::now().to_string("/tmp/screenshot-%Y-%m-%d-%H-%M-%S.png"sv);
+    }
```

Comment:
```suggestion
    if (edit_image)
        output_path = Core::DateTime::now().to_string("/tmp/screenshot-%Y-%m-%d-%H-%M-%S.png"sv);
```
No braces necessary.